CONTRIBUTING.md

```diff
@@ -1,5 +1,61 @@
 # Contributing to rticonnext-examples
 
+## How to request a feature or enhacement
+
+If there is a new feature or enhacement that you would like to be added to
+the examples, feel free to open an issue in this repository and we will take
+a look of your issue as soon as possible. Please, do not forget to use a
+title and clear description about your feature or enhacement you would like
+us to include.
+
+## How to submit a change
+
+If you have any change to submit do not hesitate in opening a Pull Request.
+Before you submit the Pull Request, please be sure you are using a correct
```

Comment:
What is 'a correct code style/format'? How people can know that? If "a correct code style/format" is what we explain in the following paragraphs, we can mention this.
```suggestion
Before you submit the Pull Request, please be sure you are using the correct
code style/format that is explained in the following paragraphs.
```

Reply:
we can also say 'that is explained below'.